unitex/src/fr/umlv/unitex/config/Config.java

```diff
@@ -71,6 +76,13 @@
  * @author Sébastien Paumier
  */
 public class Config {
+	
+	private static final int PORT = 9999;
```

Comment:
I've reengineered to use jUnique instead. This fixes some of the issues below.
Are you happy to incorporate a dependency on jUnique?
If not I can reengineer the existing solution to use a system generated port and store the port number in a local file. The isRunning() method would check this file to retrieve the port number.